packages/versioning/src/versioning.ts

```diff
@@ -352,32 +352,24 @@ interface VersionProjections {
   projections: ProjectionApplication[];
 }
 
-const versionIndex = new Map<Version, Map<NamespaceType, Version>>();
-
-function indexVersions(resolutions: VersionResolution[]) {
-  versionIndex.clear();
-  for (const resolution of resolutions) {
-    for (const version of resolution.versions.values()) {
-      versionIndex.set(version, resolution.versions);
-    }
-  }
-}
-
 export function buildVersionProjections(
   program: Program,
   rootNs: NamespaceType
 ): VersionProjections[] {
   const resolutions = resolveVersions(program, rootNs);
-  indexVersions(resolutions);
   return resolutions.map((resolution) => {
-    const projections = [...resolution.versions.entries()].map(([ns, version]) => {
-      return {
-        scope: ns,
-        projectionName: "v",
-        arguments: [version.enumMember],
-      };
-    });
-    return { version: resolution.rootVersion?.value, projections };
+    // const projections = [...resolution.versions.entries()].map(([ns, version]) => {
+    //   return;
+    // });
+    return {
+      version: resolution.rootVersion?.value,
+      projections: [
+        {
+          projectionName: "v",
+          arguments: [resolution.versions as any],
```

Comment:
@bterlson a few things here:
1. we don't allow non primitive types in the TS type
2. if you bypass it produce a type: "Object" with properties being what you passed but there is no type for it